agreements.md

```diff
@@ -0,0 +1,105 @@
+
+The Bug Zoo is a useful resource that allows experienced staff members to quickly redirect colleagues to an explanation when they see a bug that they had already dealt with at some point in the past. 
+
+When a Reef Technologies developer encounters a bug that fits this description, they should propose changes to the [Bug Zoo repository](https://github.com/reef-technologies/python-review-zoo/) and describe the bug and an appropriate solution.
+
+## Fast track decisions via Slack instead of standard Sociocracy approach
+
+At Reef Technologies, we mostly make decisions in our weekly Sociocracy meetings. The standard Sociocracy framework for proposing changes is a useful tool, but we admit that it can be time-consuming. Additionally, in some cases, a staff member may not want to wait for the Sociocracy call.
+
+The fast-track approach for decision-making allows staff members to propose “small changes”, such as minor operational improvements on the #sociocracy channel on Slack. Everyone is invited to cast their votes through upvoting (:thumbsup:) or downvoting (:thumbsdown:)  the message.
+
+## 24h format
+
+Some staff members mentioned that the 12-hour clock led to unnecessary confusion. As we are an international company and we live in different time zones, it can be doubly problematic. Thus, we decided to always use the 24-hour standard (the so-called “military time”) to minimize misunderstandings.
+
+## Joking in company meetings
+
+We want the company environment to be friendly and allow for joking around in good humor. However, we realize that due to cultural differences or a slip of the tongue, someone may make a joke that is offensive to another staff member. If such a situation occurs, we take the following steps:
+- The person who feels that something offensive has been said reacts by saying: “Hey, that was inappropriate”.
+- The author of the offensive comment says: “I’m sorry”, no matter their opinion on whether what they said was actually offensive or not.
+- Nobody else speaks up or tries to defend either party.
+- We take a few seconds to reflect on what just happened and then immediately return to whatever topic we were discussing before.
+
+This solution is meant to allow the team to auto-calibrate and recognize what comments other members may see as offensive without unnecessary discussions on delicate subjects.
+
+If the offensive comment is seen as severe and above the grade of this agreement, it will be raised to the HR circle to decide on further steps. Fortunately, so far, there have been no such situations.
+
+## Audio/video processing in Zoom calls
+
+It is not allowed to use audio or video processing in company Zoom calls. The only exception to this rule is the active noise cancellation function, which staff members may use freely.
+
+## Hardware co-funding
```

Comment:
I thought this one is going to be moved to the legal agreement

Reply:
eventually